driver_path.py

```diff
@@ -0,0 +1,17 @@
+import configparser
+config = configparser.ConfigParser()
+config.read('config.ini')
+
+
+# mailbox_list = ['lkml', 'opensuse-kernel', 'opensuse-features', 'opensuse', 'opensuse-bugs', 'opensuse-factory', 'sakai-devel']
+class driver_path_class:
```

Comment:
It is appropriate to name this class as Config which extends the the [ConfigParser](https://docs.python.org/3/library/configparser.html) class. We get all the dictionary notation for free; over and top of that, in the constructor, the necessary fields can be initialized. The approach of creating the necessary variables as fields of the config class is very helpful.

Other classes and modules can import the Config class, create an object named config and continue to use it the way you are using the `path_ob` variable.

Reply:
The module name can be `mlcatconfig` and the module itself can be placed in lib/ directory. Since all the drivers import it, the lib/ location should not pose any problems.